appcues/src/main/java/com/appcues/trait/appcues/TooltipsPaths.kt

```diff
@@ -163,89 +198,166 @@ private fun getTooltipPointerPath(
     animationSpec: AnimationSpec<Float>,
 ): Path {
     val density = LocalDensity.current
+    val cornerRadiusPx = with(density) { containerDimens.cornerRadius.toPx() }
 
-    val pointerOffsetX = calculatePointerXOffset(
+    val (pointerOffsetX, verticalTipOffset) = calculatePointerXOffset(
         containerDimens = containerDimens,
         tooltipSettings = tooltipSettings,
         pointerOffsetX = with(density) { tooltipSettings.pointerOffsetX.value.toPx() },
-        cornerRadius = with(density) { containerDimens.cornerRadius.toPx() },
+        cornerRadiusPx = cornerRadiusPx,
+    )
+
+    val (pointerOffsetY, horizontalTipOffset) = calculatePointerYOffset(
+        containerDimens = containerDimens,
+        tooltipSettings = tooltipSettings,
+        pointerOffsetY = with(density) { tooltipSettings.pointerOffsetY.value.toPx() },
+        cornerRadiusPx = cornerRadiusPx,
+    )
+
+    val tipLength = animateFloatAsState(
+        targetValue = when (tooltipSettings.tooltipPointerPosition) {
+            Top, Left -> -tooltipSettings.pointerLengthPx
+            Bottom, Right -> tooltipSettings.pointerLengthPx
```

Comment:
can these two case be combined?

Reply:
they all do the same thing right? return `tooltipSettings.pointerLengthPx`? Seemed like line 219 and 220 had identical result

Reply:
line 219 is negativa value

Reply:
ah! thank you